GlossaryOfTerms.ttl

```diff
@@ -38,7 +39,7 @@ sourceDAV:
     schema:description """Description of OpenLink Software Glossary of Terms."""@en ;
     schema:author <http://www.openlinksw.com/#this> ;
     schema:dateCreated "2014-01-03T13:00:00-05:00"^^xsd:dateTime ;
-    schema:dateModified "2021-10-14T16:45:00-05:00"^^xsd:dateTime ;
+    schema:dateModified "2021-10-12T10:35:00-05:00"^^xsd:dateTime ;
```

Comment:
This update was applied in March 2022, but the `schema:dateModified` was set to October 2021?  This appears to be part of the effective revert of the [previous commit from me](https://github.com/OpenLinkSoftware/glossaries/commit/cac9b5a2767189b21c2e144ef01b492d796ad95b) (applied by @openlink).

Reply:
Fixed